VideoLocker/AndroidManifest.xml

```diff
@@ -93,28 +93,27 @@
 
         <activity
             android:name=".view.CourseDiscussionTopicsActivity"
+            android:configChanges="orientation|keyboardHidden|screenSize"
```

Comment:
I don't think we need to block Activity reinstantiation, as long as we're ensuring that the retained Fragments are not holding on to a reference of the initial Activity instance. Even the existing video player Activities should technically be able to handle reinstantiation.